src/main/java/com/pubnub/api/enums/MessageType.java

```diff
@@ -0,0 +1,39 @@
+package com.pubnub.api.enums;
+
+import com.pubnub.api.PubNubRuntimeException;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.pubnub.api.builder.PubNubErrorBuilder.PNERROBJ_UNKNOWN_MESSAGE_TYPE;
+
+public enum MessageType {
+
+    MESSAGE01(null),
+    MESSAGE02(0),
+    SIGNAL(1),         //this value is not expected in history call
```

Comment:
We have a type representing what can and what cannot be expected in history call already so this comments are not needed

Reply:
ok, makes sense.